### ISSUE TITLE
Decrease rate

### ISSUE THREAD

**Issue by CoftheTMP** [orpheuslummis/AstaVerde#123]
I wrote in another QA response, the reduction time is something like a dollar a day rather than say 5 or 10 dollars a day. I've gone into the admin panel and although theres a possibility to change the flags for a long period to sale and short period to sale, there isn't the daily decrease rate to change. 

I wonder if that was what you meant in Set Base Price, because after the very first auction I wouldn't intend to touch that seeing as each new auction hopefully creates the price from the previous price and the Auction Threshold outcome.

In Short:
Please add the option to change the per day decrease in USDC.

**Comment by orpheuslummis** [comment]
Confirm is if this is still happening in the latest build @CoftheTMP
